app/src/main/res/values/styles.xml

```diff
@@ -5,9 +5,8 @@
 
         <item name="colorAccent">@color/accent</item>
         <item name="colorPrimaryDark">@color/background</item>
-        <item name="android:colorBackground">@android:color/black</item>
         <item name="android:textColor">@android:color/white</item>
-        <item name="android:textColorSecondary">@android:color/darker_gray</item>
+        <item name="android:textColorSecondary">#FAE2A4</item>
```

Comment:
you use `#FAE2A4` literally here, but you also set `@color/text_light` to this value. you could use the variable here. also, `android:textColor` a few lines down is set to the same color, idk if that's intentional